bin/zopen-importenvs

```diff
@@ -37,19 +38,18 @@ then
 fi
   echo "final depsPath = $depsPath"
 
-# if the number of arguments passed to script is 0,
-# then , the ports are fetched from $HOME/zopen/prod and $HOME/zopen/boot directory
-# then its env files are fetched and sourced, if they are not already sourced.
-if [ $# -eq 0 ]
-then
-     echo "Sourcing projects from $HOME/zopen/prod and $HOME/zopen/boot"
-     
-     # arrOfDirs, is an array which holds the names of dirs whose env is set
+
+sourceEnvFromPath () {
+    echo "sourceEnvFromPath enter, called with arg - $1"
```

Comment:
also looks like debug output, can it be removed?

Reply:
yes, I have removed it. Thank you.